### PR TITLE
Fix problem with health_config in operator

### PR DESCRIPTION
**Pull request by aljesusg**
Fix https://github.com/kiali/kiali/issues/3373

### CODE REVIEW

**Review by jmazzitelli — APPROVED** [review]
I'm going to approve, but see my comment in the master PR for some additional tests you should consider adding:

https://github.com/kiali/kiali-operator/pull/183#pullrequestreview-519603421

**Comment by hhovsepy** [comment]
I see this error in kiali-operator pod:
[kiali-operator-log.txt](https://github.com/kiali/kiali-operator/files/5459013/kiali-operator-log.txt)

**Review by jmazzitelli — APPROVED** [review]
I'm going to ignore the fact that this uses list indentation not consistent with everything else in the operator repo, because I'm just happy more molecule tests are being added :)

File: molecule/config-values-test/converge.yml [hunk 1]
```diff
@@ -79,6 +101,19 @@
...
+        that:
+          - kiali_configmap.health_config.rates | length == 2
+          - kiali_configmap.health_config.rates[0].namespace == "bookinfo"
+          - kiali_configmap.health_config.rates[0].tolerance | length == 2
+          - kiali_configmap.health_config.rates[0].tolerance[1].failure == 10
+          - kiali_configmap.health_config.rates[0].tolerance[1].code == "[45]\\d[^\\D4]"
+          - kiali_configmap.health_config.rates[1].namespace == ".*"
+          - kiali_configmap.health_config.rates[1].tolerance | length == 2
+          - kiali_configmap.health_config.rates[1].tolerance[0].degraded == 30
+          - kiali_configmap.health_config.rates[1].tolerance[1].code == "[5]\\d\\d"
```
Comment (jmazzitelli) [inline molecule/config-values-test/converge.yml:115]: lists should be indented at the same level, not doubly-indented, to maintain consistency. This is minor, I'm approving anyway. One day I'll go in and reformat this and the `custom_health_config` definition above. :) 

```suggestion
        that:
        - kiali_configmap.health_config.rates | length == 2
        - kiali_configmap.health_config.rates[0].namespace == "bookinfo"
        - kiali_configmap.health_config.rates[0].tolerance | length == 2
        - kiali_configmap.health_config.rates[0].tolerance[1].failure == 10
        - kiali_configmap.health_config.rates[0].tolerance[1].code == "[45]\\d[^\\D4]"
        - kiali_configmap.health_config.rates[1].namespace == ".*"
        - kiali_configmap.health_config.rates[1].tolerance | length == 2
        - kiali_configmap.health_config.rates[1].tolerance[0].degraded == 30
        - kiali_configmap.health_config.rates[1].tolerance[1].code == "[5]\\d\\d"
```

**Review by jmazzitelli — APPROVED** [review]

File: molecule/config-values-test/converge.yml (hunk 1 above)
Comment (aljesusg) [reply molecule/config-values-test/converge.yml]: Fixed ;)

Comment (jmazzitelli) [reply molecule/config-values-test/converge.yml]: > and the `custom_health_config` definition above

but not this :) (i.e. lines 55-69)

**Review by hhovsepy — APPROVED** [review]
Verified.

**Pull request merged**